lib/stream.js

```diff
@@ -12,3 +14,38 @@ Stream.PassThrough = require('_stream_passthrough');
 
 // Backwards-compat with node 0.4.x
 Stream.Stream = Stream;
+
+// Internal utilities
+try {
+  Stream._isUint8Array = process.binding('util').isUint8Array;
```

Comment:
@joyeecheung The thing is that the streams modules can’t really unconditionally rely on internal modules because they are also exported as https://github.com/nodejs/readable-stream. @mscdex expressed a preference for having this code polyfilled on the `readable-stream` side and I agree, but ultimately that’s @nodejs/streams’ call …

(Please let me know if I’m misunderstanding you!)